src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -392,15 +408,15 @@ class DetailsCard extends React.Component {
           break
 
         case 'cpu':
-          if (isNumber(value) && value > 0 && value <= this.props.maxNumOfVmCpus) {
+          if (isNumber(value) && value > 0) {
```

Comment:
please add a comment that we allow here invalid cpu to be saved in state to preserve the value after cluster change